README.md

```diff
@@ -171,6 +173,45 @@ There is no standard range for touch values aside from requirement to have x axi
 | Offset | Length | Type | Meaning |
 | ------ | ------ | ---- | ------- |
 | 0 | 1 | Unsigned 8-bit | Is touch active (`1` if active, else `0`) |
-| 1 | 1 | Unsigned 8-bit | Touch id (should be the same for one continious touch) |
+| 1 | 1 | Unsigned 8-bit | Touch id (should be the same for one continuous touch) |
 | 2 | 2 | Unsigned 16-bit | Touch X position |
 | 4 | 2 | Unsigned 16-bit | Touch Y position |
+
+## (Optional) Information about controller motors
+
+Ask/Reply how many motors in controller for some port.
+
+However, "DualShock" means there are two motors, so this interface does not need to be implemented. It can be agreed that id `0` is the left motor and id `1` is the right motor.
```

Comment:
I don't think that making this optional to implement is a good idea for a few reasons:

1. While originally protocol was indeed designed only with DS3/4 in mind, it's used for many different controllers these days
2. Querying this interface should allow client to test if server supports rumble feature. As such, no response from this interface = no sending rumble commands to server that does not support them.